inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java

```diff
@@ -429,34 +429,34 @@ private void keys(final Character c, final Node n, final StringBuilder b,
         {
             private final Character _c;
```

Comment:
If you keep the underscore here, you make the refactoring inconsistent. You should get rid of all leading underscores.